<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>requestData_dialog</class>
 <widget class="QDialog" name="requestData_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Exportando parcela</string>
  </property>
  <layout class="QVBoxLayout" name="dialogLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="parcela_lbl">
       <property name="text">
        <string>Parcela</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="area_lbl">
       <property name="text">
        <string>Area (m²)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="area_dsb">
       <property name="maximum">
        <double>9999.989999999999782</double>
       </property>
       <property name="singleStep">
        <double>5.000000000000000</double>
       </property>
       <property name="value">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="diaEdicion_chb">
       <property name="text">
        <string>Dia de edición</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDateEdit" name="diaEdicion_date">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="archivo_lbl">
       <property name="text">
        <string>Archivo a exportar</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="archivo_btn">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="parcela_tbx">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>0822805DF1802S</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>area_dsb</tabstop>
  <tabstop>archivo_btn</tabstop>
  <tabstop>diaEdicion_chb</tabstop>
  <tabstop>diaEdicion_date</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>requestData_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>requestData_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>diaEdicion_chb</sender>
   <signal>toggled(bool)</signal>
   <receiver>diaEdicion_date</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
